pkg/reconciler/instances/istio/actions/performer.go

```diff
@@ -219,6 +233,19 @@ func (c *DefaultIstioPerformer) PatchMutatingWebhook(context context.Context, ku
 	return nil
 }
 
+func (c *DefaultIstioPerformer) selectWebhookToPatch(context context.Context, candidatesNames []string, clientSet clientgo.Interface) (*v1.MutatingWebhookConfiguration, error) {
```

Comment:
I would rather call this function: `selectFirstMatchingWebhookFromCandidates` - we don't need to mention patching here